src/scripts/main.js

```diff
@@ -1,3 +1,151 @@
 'use strict';
 
-// write code here
+const thead = document.querySelectorAll('thead tr th');
```

Comment:
The `thead` variable is initialized using `querySelectorAll`, which returns a NodeList. When using it in the `forEach` loop, it works fine, but when accessing it by index (e.g., `thead[columnIndex]`), ensure that it behaves as expected. Consider converting it to an array if you need array-specific methods.